src/System.Runtime.Extensions/tests/System/IO/StreamWriterTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class StreamWriterTests
```

Comment:
The test will have to wait then as they won't reference the fixed assembly. Or is there a way around that?